draft-ietf-quic-transport.md

```diff
@@ -4196,10 +4189,7 @@ processing a Retry packet; {{packet-0rtt}} contains more information on this.
 
 A server acknowledges the use of a Retry packet for a connection using the
 original_connection_id transport parameter (see
-{{transport-parameter-definitions}}).  If the server sends a Retry packet, it
-MUST include the value of the Original Destination Connection ID field of the
-Retry packet (that is, the Destination Connection ID field from the client's
-first Initial packet) in the transport parameter.
+{{transport-parameter-definitions}}).
```

Comment:
This could be more concrete, e.g. `by setting the original_connection_id transport parameter to the Destination Connection ID from the client's first Initial packet`.

Reply:
I agree with @marten-seemann here. The server MUST set the transport parameter. You're removing normative language here.

Reply:
Good point, fixed.